templates/projects/index.html

```diff
@@ -6,165 +6,143 @@
+        <p>The following projects are covered by Canonical’s contributor agreement. 
+          If you want to contribute to any of the projects below, please contact the project contacts listed in the third column. In order to contribute, you need to sign Canonical’s contributor agreement.
+        </p>
+      </div>
     </div>
-  </div>
-</section>
+  </section>
 
-<section class="p-strip--light is-deep">
-  <div class="row">
-    <div class="col-12">
-      <h2 class="p-muted-heading">Some projects we care about</h2>
+  <section>
+    <div class="u-fixed-width">
+        <h2 class="p-muted-heading">Some projects we care about</h2>
+        <hr class="p-rule--highlight">
     </div>
-  </div>
-  <div class="row">
-    <div class="col-3">
-      <div class="p-card">
-        <header class="p-card__header">
-          <img src="https://assets.ubuntu.com/v1/41391002-image-project-ubuntu-icon.png" alt="Ubuntu" />
-        </header>
-        <h3>Ubuntu</h3>
+    <div class="row p-section">
+      <div class="col-6 col-medium-3">
+        <h3 class="p-heading--2"><a href="https://ubuntu.com">Ubuntu</a></h3>
         <p>Ubuntu is one of the world's biggest and most influential open source projects. With new features continually being developed, there is plenty of opportunity to get involved.</p>
-        <footer class="p-card__footer">
-          <a href="https://discourse.ubuntu.com/t/contribute/26">Contribute</a>
-        </footer>
-      </div>
-      <div class="p-card">
-        <header class="p-card__header">
-          <img src="https://assets.ubuntu.com/v1/2fbd96a8-image-project-mir-icon.png" alt="Mir" />
-        </header>
-        <h3>Mir</h3>
-        <p>A cross-platform client interface needs a cross-platform graphics stack. Mir is the next generation display server used by Ubuntu for its mobile and desktop form factors.</p>
-        <footer class="p-card__footer">
-          <a href="https://mir-server.io/">Learn more</a>
-        </footer>
+        <p><a href="https://discourse.ubuntu.com/t/contribute/26">Contribute</a></p>
       </div>
-      <div class="p-card">
-        <header class="p-card__header">
-          <img src="https://assets.ubuntu.com/v1/045fe3a2-maas-icon.png" alt="MAAS" />
-        </header>
-        <h3>MAAS</h3>
-        <p>Installing and deploying cloud computing at datacentre scale is often complicated and time consuming. Metal as a Service (MAAS) lets you treat physical servers like virtual machines in the cloud, turning your bare metal into an elastic, cloud-like resource.</p>
-        <footer class="p-card__footer">
-          <a href="https://maas.io">Learn more</a>
-        </footer>
+      <div class="col-6 col-medium-3">
+        <div class="p-image-wrapper">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/38ce93a4-Frame 2816.jpg",
+            alt="",
+            width="1620",
+            height="969",
+            hi_def=True,
+            loading="auto|lazy"
```

Comment:
```suggestion
            loading="auto"
```
It can be one or the other, but not both. Since this is the first (only) image on this page it should be auto